README.md

```diff
@@ -36,3 +36,18 @@ ghci> let trMap = delete @Int typeRepMap
 ghci> member @Int trMap
 False
 ```
+
+## Benchmarks
+
+Tables below contain comparision with `DMap TypeRep` of `lookup` operation
```

Comment:
Probably mention that it's for 10 lookups?

Reply:
Why are we testing only 10? Let's try 1000.

Reply:
@int-index , that was done because I don't know how to write such a function, in its type it should be 1000 `Proxy`s 😞 
Currently it looks like this:

```haskell
tenLookups :: TypeRepMap (Proxy :: Nat -> *)
           -> ( Proxy 10, Proxy 20, Proxy 30, Proxy 40
              , Proxy 50, Proxy 60, Proxy 70, Proxy 80
              )
```